data/ecosystems/e/ergo.toml

```diff
@@ -2,48 +2,79 @@
 title = "Ergo"
 
 sub_ecosystems = [
-  "anetaBTC",
-  "Azorus",
-  "BlobsTopia",
-  "ChainCash",
-  "Comet",
-  "Dexy",
-  "Duckpools",
-  "Ergo Auction House",
-  "Ergo Developer Tooling",
-  "Ergo Education",
-  "Ergo Hackathons",
-  "Ergo Miner Tooling",
-  "Ergo Names",
-  "Ergo Raffle",
-  "Ergo Stats and Analysis",
-  "Ergode",
-  "ErgoMixer",
-  "ErgoPad",
-  "ErgoTeam",
-  "EXLE",
-  "FleetSDK",
-  "Grand Gambit",
-  "GuapSwap",
-  "Kiosk",
-  "LiliumErgo",
-  "Lithos Protocol",
-  "Minotaur",
-  "Nautilus",
-  "NightOwl Casino",
-  "PaideiaDAO",
-  "Rosen Bridge",
-  "SAFEW",
-  "Satergo",
-  "SentientFoundation",
-  "SigmaFi",
-  "SigmaUSD",
-  "SkyHarbor",
-  "Spectrum Finance",
-  "Swamp Audio",
-  "ZenGate Global",
-  "ZK Treasury",
-]
+    # Exchanges
+    "Spectrum Finance",
+    "anetaBTC",
+
+    # Launchpad
+    "ErgoPad",
+
+    # Stablecoins
+    "ChainCash",
+    "Dexy",
+    "SigmaUSD", 
+
+    # Fundraising
+    "Ergo Raffle", 
+
+    # Mining
+    "Lithos Protocol",
+    "GuapSwap", 
+    "Ergo Miner Tooling",
+    
+    # Lending
+    "EXLE",
+    "Duckpools",
+    "SigmaFi", 
+
+    # Gambling
+    "NightOwl Casino",
+    "BlobsTopia",
+    "Grand Gambit",
+
+    # Degen
+    "HodlcoinERG",
+
+    # Wallets
+    "Nautilus",
+    "Minotaur",
+    "Satergo",
+    "SAFEW", 
+
+    # Privacy
+    "ErgoMixer",
+    "ZK Treasury",
+
+    # NFTs 
+    "SkyHarbor",
+    "Ergo Auction House",
+
+    # Dev
+    "FleetSDK",
+    "Kiosk",
+    "Azorus",
+    "Ergo Education", 
+    "Ergo Developer Tooling",
+    "Ergo Hackathons", 
+
+    # DAO 
+    "PaideiaDAO",
+    "ErgoTeam",
+    "Swamp Audio",
+
+    # Layer 2s / Sidechains / Bridges
+    "SentientFoundation",
+    "Rosen Bridge",
+
+    # Misc 
+    "ZenGate Global",
+    "Ergode",
+    "Ergo Names",
+    "LiliumErgo",
+    "Comet", 
+    "Ergo Stats and Analysis"
+
+    ]
```

Comment:
please keep the original alphabetical sorting as we have it as a standard in all the other toml files to make it easier to visually search for ecosystem names

Reply:
fixed